docs/terms-policies/governance.md

```diff
@@ -0,0 +1,267 @@
+| [dandi/dandi-about](https://github.com/dandi/dandi-about) | [@dandi/dandi-docs-maintainers](https://github.com/orgs/dandi/teams/dandi-docs-maintainers) |
+| [dandi/dandi-docs](https://github.com/dandi/dandi-docs) | [@dandi/dandi-docs-maintainers](https://github.com/orgs/dandi/teams/dandi-docs-maintainers) |
+| [dandi/example-notebooks](https://github.com/dandi/example-notebooks) | [@dandi/dandi-docs-maintainers](https://github.com/orgs/dandi/teams/dandi-docs-maintainers) |
+
+### 5.4 Project Leadership
+- Current leadership team:
+    - [Satrajit Ghosh](https://satra.cogitatum.org/) ([@satra](https://github.com/satra))
+    - [Yaroslav O. Halchenko](https://centerforopenneuroscience.org/whoweare) ([@yarikoptic](https://github.com/yarikoptic))
+
+- Responsibilities:
+    - Approve or amend governance document and Code of Conduct.
+    - Strategic project oversight.
+    - Resolve escalated disputes.
+    - Approve major architectural shifts.
+    - Oversee risk, sustainability, funding alignment.
+
+### 5.5 DANDI Archive Administrators
+
+Responsibilities:
+- Monitor service health and performance.
+- Manage user accounts and permissions.
+- Respond to data access requests and issues.
+
+Administrators:
+- @satra, @yarikoptic, @waxlamp, @mvandenburgh, @jjnesbitt, @danlamanna, @brianhelba, @kabilar, @asmacdo, @bendichter, @candleindark, @CodyCBakerPhD.
+
+## 6. Decision-Making Model
+
+### 6.1 Roadmap
+- Project targets are discussed during the biweekly Engineering Core and Scientific Core meetings.
+- Project Leadership provides guidance on prioritization of targets.
+- Public notes of these meetings are available on [Google Drive](https://drive.google.com/drive/folders/1-jXLpcrh3L650FiZyTFgcs096nZjO2C3).
+
+### 6.2 Consensus Process
+1. Open a GitHub issue describing the bug/feature request, context, and possible solutions.
+2. For major architectural changes, create a design document and submit as a PR for discussion and refinement.
+   For reference, see [previous design documents](https://github.com/dandi/dandi-archive/tree/master/doc).
+3. Collect feedback during a 7-day review period.
+4. Summarize consensus in the GitHub issue and resolve all suggestions in the design document.
+5. Implement via pull request(s) referencing the proposed design.
+
+### 6.3 Conflict of Interest
+- Participants should disclose direct commercial interest in a technology choice.
+- Conflicted member recuses themselves from final decision phase.
+
+## 7. Pull Request Workflow
+
+### 7.1 Pull Request Requirements
+- Link the associated issue.
+- Add a clear description (problem, approach, alternatives considered).
+- Major architectural changes require a design document.
+- Add or update tests.
+- Update documentation.
+- Ensure CI passes.
+- Large pull requests should be split unless justified.
+- No introduction of unreviewed secrets or credentials.
+- Verified provenance for large binary additions (discouraged in code repos).
```

Comment:
This is fine as a kind of illustration of the sort of thing we expect from PR contributors, but some of it is so vague as to not be very useful (what is "verified provenance" for a large binary addition, and if it's discouraged, why mention it?)

Generally, the roles and requirements sections feel vague in this way. Maybe we can add language explaining that such lists "include, but are not limited to" the mentioned items.